api/src/main/java/org/pmiops/workbench/api/OfflineUserController.java

```diff
@@ -257,6 +258,17 @@ public OfflineUserController(
     return ResponseEntity.noContent().build();
   }
 
+  @Override
+  public ResponseEntity<Void> synchronizeUserAccess() {
+    userService.getAllUsers().stream()
+        .forEach(
+            user ->
+                // This will update the access tiers for a given user, based on compliance rules
+                // If a user is no longer compliant it will remove them from an access tier
+                userService.updateUserWithRetries(Function.identity(), user, Agent.asAdmin(user)));
```

Comment:
Agent (3rd param) here should be service, since it's being done offline. As currently written, this will log that the user updated their own access level while acting as an admin